src/utils/displayTo.ts

```diff
@@ -1,33 +1,88 @@
+    (acc, next) => (next.breakpoints ? { ...acc, [next.screen]: formatQuery(next.breakpoints) } : acc),
+    {},
+  );
 }
 
-export function getMediaQueries(breakpoints: Breakpoints) {
-  if (!isBreakpoint(breakpoints, 'medium') || !isBreakpoint(breakpoints, 'large')) {
-    throw new Error('Invaild breakpoints');
+// create max-width media query including screen size
+export function displayUpTo(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
   }
+  const query = formatQuery({ next: screenBreakpoints.next });
+  return formatMedia(query);
+}
 
-  return {
-    small: `(max-width: ${breakpoints.medium - 1}px)`,
-    smallAndMedium: `(max-width: ${breakpoints.large - 1}px)`,
-    medium: `(min-width: ${breakpoints.medium}px) and (max-width: ${breakpoints.large - 1}px)`,
-    mediumAndLarge: `(min-width: ${breakpoints.medium}px)`,
-    large: `(min-width: ${breakpoints.large}px)`,
-  };
+// create min-width media query including screen size
+export function displayFrom(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
+  }
+  const query = formatQuery({ prev: screenBreakpoints.prev });
+  return formatMedia(query);
 }
 
 export function displayTo(screen: ScreenSize | string): ThemedCssFunction<any> {
-  const query = getMediaQueries(breakpoints)[screen] || screen;
-  const result: ThemedCssFunction<any> = (strings: any, ...interpolations: Array<any>) =>
+  const screenBreakpoints = getScreenSizeBreakpoints(screen as ScreenSize, breakpoints);
+  const query = screenBreakpoints ? formatQuery(screenBreakpoints) : screen;
+  return formatMedia(query);
+}
+
+// create media query based on previous and next breakpoints
+export function formatQuery(breakpoints: { prev?: number; next?: number }) {
+  const { next, prev } = breakpoints;
+  if (prev === undefined && next === undefined) {
+    throw new Error('Invaild breakpoints');
+  }
+
+  const queries = [];
+  if (prev !== undefined) {
+    queries.push(`(min-width: ${prev}px)`);
+  }
+  if (next !== undefined) {
+    queries.push(`(max-width: ${next - 1}px)`);
+  }
+  return `${queries.join(' and ')}`;
+}
+
+export function getScreenSizeBreakpoints(
+  screen: ScreenSize,
+  breakpoints: Breakpoints,
+): { next?: number; prev?: number } | undefined {
+  switch (screen) {
+    case 'small':
+      return { next: breakpoints.medium };
+    case 'smallAndMedium':
+      console.warn("'smallAndMedium' screen size has been deprecated since 0.8.0. Use 'displayUpTo' function.");
+      return { next: breakpoints.large };
+    case 'medium':
+      return { prev: breakpoints.medium, next: breakpoints.large };
+    case 'mediumAndLarge':
+      console.warn("'mediumAndLarge' screen size has been deprecated since 0.8.0. Use 'displayFrom' function.");
```

Comment:
`Use 'displayFrom/displayUpTo' function.`. I'm not sure if it's necessary. Please remove.

Reply:
I think it's necessary because when you say that something is deprecated you should provide an alternative way.

Reply:
In addition, extra-information has never been a problem.

Reply:
Agree 😀, but see an example:

```ts
displayFrom('mediumAndLarge');
```
then console warning will show me the warning:
```bash
"'mediumAndLarge' screen size has been deprecated since 0.8.0. Use 'displayFrom' function."
```

Do you think it's a developer-friendly error message if I've already used `displayFrom` function?

Reply:
Updated message